src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("mate_user");
+        user.setAge(18);
+        user.setPassword("MyCatNameDin");
+        user.setId(713298L);
+    }
+
+    @AfterEach
+    void removeStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_userValid_Ok() {
```

Comment:
all of your tests are just exact duplicates, changing their name doesn't mean that they will work in another way. 
You should not have an exception thrown when you are passing a valid user, that is the whole point of the tests.
In `ok` test case you should check whether the user `id` has been set.
And you should change test user fields according to your test name, e.g. 
```
@Test
    void register_loginNull_notOk() {
        user.setLogin(null);
        assertThrows(RuntimeException.class, () -> registrationService.register(user));
    }
```
this will be the same user that you have initialized in the `setUp()` method but with null login